CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+
+# Contributing to Razor Network Datasources
+If you wish to propose a new price feed to be added to the datasources used by the protocol, please follow the steps below:
+
+1. Start by forking the repository and creating your branch from `master`.
+2. Add the new job to `mainnet/jobs.json` and update `mainnet/collections.json` as needed with the correct jobIDs. (an example for `testnet` is given below)
+    - For example, adding a new `ETH/USD` job to an existing collection for testnet:
+```json
+[
+  {
+    "weight": 1,
```

Comment:
Please explain what each attribute is (weight, power, selectorType, name, selector, URL, etc) and illustrate why its important.

Reply:
noted

Reply:
resolved here [6b4dd2c](https://github.com/razor-network/datasources/pull/55/commits/6b4dd2c39021acbfdeaa19e752379cd5d63f9345)